wicket-core/src/main/java/org/apache/wicket/ajax/RemoteFunctionCallUtils.java

```diff
@@ -0,0 +1,43 @@
+package org.apache.wicket.ajax;
+
+import com.github.openjson.JSONObject;
+import org.apache.wicket.ajax.json.JSONFunction;
+import org.apache.wicket.ajax.json.JsonUtils;
+import org.apache.wicket.util.string.Strings;
+import org.junit.platform.commons.util.StringUtils;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class RemoteFunctionCallUtils
+{
+	public static String createFunctionJsonString(CharSequence functionName, Object ... args)
+	{
+		return "{\"func\":" + wrapAndEscape(functionName) + "," +
+				"\"args\":[" +
+				Arrays.stream(args).map(RemoteFunctionCallUtils::wrapAndEscape).collect(Collectors.joining(",")) +
+				"]}";
+	}
+
+	/**
+	 * Currently works only with primitives
+	 * @param arg
+	 * @return
+	 */
+	public static String wrapAndEscape(Object arg)
+	{
+		if (arg instanceof Number || arg instanceof Boolean)
+		{
+			return String.valueOf(arg);
+		} else if (arg instanceof JSONObject) {
+			return arg.toString();
+		} else
```

Comment:
Formatting + other `JSON*` type for ex. `JSONArray`

Reply:
What's up with formatting?

Reply:
curly braces should be on separate line i.e.
```
}
else if (arg instanceof JSONObject)
{
```
not
```
} else if (arg instanceof JSONObject) {
```